### ISSUE TITLE
Something is broken since yesterday (v17.4.0)

### ISSUE THREAD

**Issue by wp55** [auth0/auth0-deploy-cli#596]
I don't have a clue what is broken, but deploys aren't working anymore. I don't have any unreplaced keywords in the config files. And in case there should be some unreplaced keyswords, provide a decent error message with the name of the keyword.

![image](https://user-images.githubusercontent.com/87134268/176372562-f0f8dc04-1a50-416c-a9ef-bc2bf8b373b1.png)

**Comment by wp55** [comment]
I think it's complaining on this piece of code which is standard auth0 code in the HTML template of classic universal login. On line 23.

![image](https://user-images.githubusercontent.com/87134268/176374898-43448d9d-863c-4f3d-b499-7be66befb04c.png)

**Comment by willvedd** [comment]
@wp55 You're correct that the v7.14.0 caused this issue. It stems from a feature that was intended to catch unreplaced keywords to prevent bad deploys. We didn't realize that some of the HTML templates legitimately use the same `@@` replacement syntax as the Deploy CLI, causing a false positive. This discovery also makes me realize that the Deploy CLI, even without the validation functionality, could unintentionally replace the `@@config@@` strings in templates, causing a potentially more nefarious issue. Needless to say, the validation functionality that was added in 7.14.0 needs to be thought-out a bit more. In the meantime I've reverted it and have pushed 7.14.1 as a fix. Apologies!
